cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicasHelper.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ *
+ */
+
+package com.linkedin.kafka.cruisecontrol.model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+
+
+/**
+ * A helper class to create new {@link SortedReplicas} for cluster/broker/disk.
+ * This class provides fluent API to enable user to customize {@link SortedReplicas} by adding new selection/priority/score
+ * functions one by one.
+ *
+ * One example is as follows( to register a SortedReplica to clusterModel which includes only follower replicas, prioritize offline
```

Comment:
Why is the explanation of example is in the parentheses? Is it optional to read it to understand the example?